src/main/java/edu/kis/powp/jobs2d/drivers/adapter/BasicLineDrawerAdapter.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+/**
+ * driver adapter to drawer - Basic Line.
+ */
+public class BasicLineDrawerAdapter implements Job2dDriver {
+    private int startX = 0, startY = 0;
+    private final DrawPanelController drawPanelController = DrawerFeature.getDrawerController();
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.